src/pages/Governance/hooks/useSubmitStake.ts

```diff
@@ -10,22 +10,16 @@ const useSubmitStake = () => {
   } = useSubmitTransaction();
 
   async function submitStake(
-    stakingAmount: number,
+    stakingAmount: string,
```

Comment:
should be string for now, once Petra move to use ABI interface we should be able to send a number
https://aptos-org.slack.com/archives/C03N83P7QUC/p1660945235254379